src/keylessBackup/utils.ts

```diff
@@ -0,0 +1,7 @@
+// helper functions that default to false, but allows testing by mocking
+
+// TODO(ACT-771): remove when status taken from statsig
+export const getKeylessBackupGate = () => false
+
+// TODO(ACT-684, ACT-766, ACT-767): remove when status taken from redux
+export const isKeylessBackupCompleteUtil = () => false
```

Comment:
nit: I know this is going to eventually be deleted, but not sure I understand the need for the `Util` suffix

Reply:
haha I was just been lazy with naming, updated